modules/services/clipman.nix

```
@@ -0,0 +1,58 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.clipman;
+
+in {
+  meta.maintainers = [ maintainers.jwygoda ];
+
+  options.services.clipman = {
+    enable = mkEnableOption "clipman, a simple clipboard manager for Wayland";
+
+    package = mkPackageOption pkgs "clipman" { };
+
+    systemd_target = mkOption {
```

Option names should, whenever possible, use camel case:

```suggestion
    systemdTarget = mkOption {
```